LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright 2017-2019 Ethereum Foundation
+Copyright (c) 2020 The Ethereum Foundation
```

Comment:
I have no idea how this works, but should we update this annually? As in, should this be 2023 now?

Reply:
As discussed, I'll update to the range of years in which changes were made to the library.